### PR TITLE
reorder venue drinks

### PR DESCRIPTION
**Pull request by SimonLab**
ref: #600

### CODE REVIEW

File: lib/cs_guide_web/controllers/brand_controller.ex [hunk 1]
```diff
@@ -152,47 +152,6 @@ defmodule CsGuideWeb.BrandController do
...
-    |> max_by_name()
-  end
-
-  @doc """
-  Returns the most common element of a list
-  ## Example
-      iex> max_by_name(["one", "two", "one"])
-      "one
-  """
-  def max_by_name([]), do: nil
```
Comment (SimonLab) [inline lib/cs_guide_web/controllers/brand_controller.ex]: Move these function to `Resources.Drink` module to be able to reuse them in venue controller

File: lib/cs_guide_web/controllers/venue_controller.ex [hunk 2]
```diff
@@ -8,7 +8,17 @@ defmodule CsGuideWeb.VenueController do
   alias CsGuide.Accounts.{User, VenueUser}
 
   @mailer Application.get_env(:cs_guide, :mailer) || CsGuide.Mailer
-  @ten_days 864000 # ten days in seconds
+  # ten days in seconds
+  @ten_days 864_000
+
+  # see issue #600
+  @order_drink_types %{ "Tonics & Mixers" => 1,
```
Comment (SimonLab) [inline lib/cs_guide_web/controllers/venue_controller.ex]: Associate integer value to drink types to make it easier to sort drinks by type

File: lib/cs_guide_web/views/venue_view.ex [hunk 3]
```diff
@@ -65,8 +65,4 @@ defmodule CsGuideWeb.VenueView do
     |> Enum.join("+")
   end
 
-  def order_drinks_by_type(drinks) do
```
Comment (SimonLab) [inline lib/cs_guide_web/views/venue_view.ex]: The drink are now sorted on the venue controller instead of the view

Comment (nelsonic) [reply lib/cs_guide_web/views/venue_view.ex]: @SimonLab makes sense. 👍

**Pull request merged**